### ISSUE TITLE
Show do not disturb phone profile

### ISSUE THREAD

**Issue by calvaris** [andyholmes/gnome-shell-extension-gsconnect#451]
Really quick, I think it could be nice to see the "do not disturb" profile status of your phone in the app. I am not even sure if KDE Connect provides that info but I thought it would be interesting to comment it here at least.

**Comment by ferdnyc** [comment]
@calvaris Device DND status isn't currently exported by KDE Connect, so it would have to be added there first for GSConnect to be able to display it locally. 

In theory they could query [`android.app.NotificationManager.getCurrentInterruptionFilter()`](https://developer.android.com/reference/android/app/NotificationManager.html#getCurrentInterruptionFilter()) and return the value in a state packet (at least, I don't know of any reason that wouldn't be possible), but there's currently no code in the Android app to check the DND state of the device.

One wrinkle is that, while the app can actively _query_ the state, there [doesn't appear to be](https://issuetracker.google.com/issues/37127718) any way for an app to listen for _changes_ to the DND state. That might actually be a real problem, as having the app actively poll the status would be inefficient and a significant drain on battery life. The bug's been open for a couple of years now, but so far no progress.

Still, if you think it would be a useful feature to have in KDE Connect (and then, presumably, in GSConnect as well, once the info's being delivered by the Android app), I'd suggest [submitting a feature request to the KDE Connect bugtracker](https://bugs.kde.org/enter_bug.cgi?product=kdeconnect). (See [the KDE Connect wiki page](https://community.kde.org/KDEConnect) first for more info, if you're not already familiar with it.)

**Comment by andyholmes** [comment]
FYI, KDE Connect has migrated to GitLab. You can file any issues here now: https://invent.kde.org/kde/kdeconnect-android

**Comment by ferdnyc** [comment]
Oh, neat! They should really update their own Wiki page. :laughing: (Which I see they've started to do, today, as this switch is apparently very recent. The "Development" section has already been changed to point to GitLab. The bug tracker link just hasn't been updated yet.)

Just finishing my "KDE Identity" (**#fancy**) registration now, so I can actually log in. 

<details>
<summary>Griping about KDE Identity follows...</summary>

Oh, look, KDE Identity:
1. Doesn't allow you to choose your own username, and forces you to choose from a pre-defined set of combinations based around your first and last name. How very Facebook of them!
1. Will let you set Gender in your profile, but only to Male, Female, or "Other". Inclusive!
1. Accepts Date of Birth, but only if you're willing to give up the _full_ date — it won't accept just a year. (If nothing else, I don't want them to have my exact DOB because I already get MORE than enough lame automated emails from different services I've signed up with, every year on that day.)

\*sigh* This is all super-awesome. I'm glad all the similar services people have built over the years had such an impact, that they chose to take the wrong lesson every single time.  /s
</details>

**Comment by kdesysadmin** [comment]
@andyholmes Please remove that comment as it is incorrect. Gitlab is being used for code management, along with internal task tracking only.

All bug reports for KDE Connect are to be reported to the KDE Bug Tracking system at https://bugs.kde.org/

**Comment by ferdnyc** [comment]
@kdesysadmin Yeah, that hadn't actually been communicated at that time (not to us, and it seems no to the KDE Connect team either) — I'd [opened an issue](https://invent.kde.org/kde/kdeconnect-kde/issues/1) in the GitLab tracker for KDE Connect and eventually we all found our way to the same page. I've updated @andyholmes' comment above to correct.

**Comment by andyholmes** [comment]
@kdesysadmin thanks for the heads up, I've also corrected our wiki page :)

@ferdnyc thanks for getting the edit here :)
